src/comodo/jaxsimSimulator/jaxsimSimulator.py

```diff
@@ -0,0 +1,465 @@
+
+    @property
+    def feet_wrench(self) -> npt.ArrayLike:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        wrenches = self.link_contact_forces
+
+        left_foot = np.array(wrenches[self._left_foot_link_idx])
+        right_foot = np.array(wrenches[self._right_foot_link_idx])
+        return left_foot, right_foot
+
+    @property
+    def base_transform(self) -> npt.ArrayLike:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        return np.array(self._data.base_transform())
+
+    @property
+    def base_velocity(self) -> npt.ArrayLike:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        return np.array(self._data.base_velocity())
+
+    @property
+    def simulation_time(self) -> float:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        return self._data.time()
+
+    @property
+    def link_contact_forces(self) -> npt.ArrayLike:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        if self._contact_model_type is ContactModelEnum.VISCO_ELASTIC:
+            raise ValueError(
+                "Link contact forces are only available for non visco-elastic contact models."
+            )
+        return self._link_contact_forces
+
+    @property
+    def total_mass(self) -> float:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        return js.model.total_mass(self._model)
+
+    @property
+    def feet_positions(self) -> tuple[npt.ArrayLike, npt.ArrayLike]:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        W_p_lf = js.frame.transform(
+            model=self._model,
+            data=self._data,
+            frame_index=self._left_footsole_frame_idx,
+        )[0:3, 3]
+        W_p_rf = js.frame.transform(
+            model=self._model,
+            data=self._data,
+            frame_index=self._right_footsole_frame_idx,
+        )[0:3, 3]
+
+        return (W_p_lf, W_p_rf)
+
+    @property
+    def com_position(self) -> npt.ArrayLike:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        return js.com.com_position(self._model, self._data)
+
+    def reset_simulation_time(self) -> None:
+        assert (
+            self._is_initialized
+        ), "Simulator is not initialized, call load_model first."
+        self._data = self._data.replace(time_ns=jnp.array(0, dtype=jnp.uint64))
+        assert self._data.time_ns == 0, "Failed to reset simulation time."
+
+    def update_contact_model_parameters(self, params: ContactsParams) -> None:
```

Comment:
Once https://github.com/ami-iit/jaxsim/pull/260 gets merged

```suggestion
    def update_contact_model_parameters(self, params: ContactParamsTypes) -> None:
```

Reply:
Done in e5573f85a31629dcb5d87ef915a1306b29ed883c